dev/com.ibm.ws.cdi.internal/src/com/ibm/ws/cdi/internal/archive/liberty/ExtensionArchiveImpl.java

```diff
@@ -29,11 +29,19 @@
 
 public class ExtensionArchiveImpl extends CDIArchiveImpl implements ExtensionArchive {
 
-    private static final TraceComponent tc = Tr.register(ExtensionArchiveImpl.class);
+    protected static final TraceComponent tc = Tr.register(ExtensionArchiveImpl.class);
```

Comment:
Why does this need to be `protected`? Usually a subclass would have its own `TraceComponent`.